autoload/rust.vim

```diff
@@ -220,6 +220,47 @@ function! s:RmDir(path)
 	silent exe "!rm -rf " . shellescape(a:path)
 endfunction
 
+" Playpen Support {{{1
+" Parts of gist.vim by Yasuhiro Matsumoto <mattn.jp@gmail.com> reused
+" gist.vim available under the BSD license, available at
+" http://github.com/mattn/gist-vim
+if globpath(&rtp, 'autoload/webapi/http.vim') !=# ''
```

Comment:
This is not compatible with lazy bundle loading with [NeoBundle](https://github.com/Shougo/neobundle.vim). An alternative approach that would work might look like

```vim
function! s:has_webapi()
    if !exists("*webapi#http#post")
        try
            function("webapi#http#post")
        catch
        endtry
    endif
    return exists("*webapi#http#post")
endfunction
```

The downside to this is it triggers a load of `webapi`, and you don't actually need it to be loaded until the user runs `:Play`. So I'd suggest doing this test at the top of `rust#Play`. Doing that also means you can unconditionally define `:Play` and have it emit an error telling the user they need to install `webapi` if it's not installed.